admin/class-metabox.php

```diff
@@ -198,11 +199,13 @@ public function localize_post_scraper_script() {
 		}
 
 		return array(
-			'translations'                => $json,
-			'keyword_usage'               => $this->get_focus_keyword_usage( $post->ID ),
-			'search_url'                  => admin_url( 'edit.php?seo_kw_filter={keyword}' ),
-			'post_edit_url'               => admin_url( 'post.php?post={id}&action=edit' ),
-			'home_url'                    => home_url( '/', null ),
+			'translations'  => $json,
```

Comment:
My suggestion is to put the lines 192-199 in a separate method like get `scraper_json`

```
private function scraper_json() {
    $file = plugin_dir_path( WPSEO_FILE ) . 'languages/wordpress-seo-' . get_locale() . '.json';
 	if ( file_exists( $file ) && $file = file_get_contents( $file ) ) 
 	    return json_decode( $file, true );
 	}		 		

    return array();
}
```